addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2715,7 +2715,9 @@ export class OdooEditor extends EventTarget {
         if (isRow && isRtl) {
             left = tableRect.right - tableUiContainerRect.x;
         } else if (isRow && !isRtl) {
-            left = elementRect.left - tableUiContainerRect.left - (isRow ? wrappedUi.clientWidth : 0);
+            left = elementRect.left - tableUiContainerRect.left - wrappedUi.clientWidth;
+        } else if (!isRow && isRtl) {
+            left = elementRect.left - tableUiContainerRect.left + wrappedUi.clientWidth;
         } else {
             left = elementRect.left - tableUiContainerRect.left - (isRow ? wrappedUi.clientWidth : 0);
         }
```

Comment:
At this point we should rearrange this to be a bit clearer, don't you think?

```suggestion
        if (isRow) {
            if (isRtl) {
                left = tableRect.right - tableUiContainerRect.x;
            } else {
                left = elementRect.left - tableUiContainerRect.left - wrappedUi.clientWidth;
            }
        } else if (isRtl) {
            left = elementRect.left - tableUiContainerRect.left + wrappedUi.clientWidth;
        } else {
            left = elementRect.left - tableUiContainerRect.left - (isRow ? wrappedUi.clientWidth : 0);
        }

Reply:
> At this point we should rearrange this to be a bit clearer, don't you think?

Sure thing :)

Reply:
GTG :+1:  @Zynton